xml/System.Reflection.Metadata.Ecma335/MetadataBuilder.xml

```diff
@@ -935,14 +1067,14 @@
         <Parameter Name="parameterList" Type="System.Reflection.Metadata.ParameterHandle" Index="5" FrameworkAlternate="netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="attributes">To be added.</param>
-        <param name="implAttributes">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="signature">To be added.</param>
-        <param name="bodyOffset">To be added.</param>
-        <param name="parameterList">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="attributes">The method attributes.</param>
+        <param name="implAttributes">The method implementation attributes.</param>
+        <param name="name">The method name.</param>
+        <param name="signature">The method signature.</param>
+        <param name="bodyOffset">Offset within the block in the PE image that stores method bodies (IL stream), or -1 if the method doesn't have a body. The final relative virtual address stored in the metadata is calculated when the metadata is serialized by adding the offset to the virtual address of the beginning of the block.</param>
+        <param name="parameterList">If the method declares parameters in the Params table, set this to the handle of the first one. Otherwise, set this to the handle of the first parameter declared by the next method definition. If no parameters are declared in the module, <see cref="M:System.Reflection.Metadata.Ecma335.MetadataTokens.ParameterHandle(System.Int32)" />(1).</param>
```

Comment:
```suggestion
        <param name="parameterList">If the method declares parameters in the Params table, set this to the handle of the first one. Otherwise, set this to the handle of the first parameter declared by the next method definition. If no parameters are declared in the module, <see langword="System.Reflection.Metadata.Ecma335.MetadataTokens.ParameterHandle(1)" />.</param>
```